src/main/java/gregtech/integration/exnihilo/recipes/ExNihiloRecipes.java

```diff
@@ -0,0 +1,234 @@
+    }
+
+    public static void registerCraftingRecipes() {
+        // Pebbles
+        ModHandler.addShapedRecipe("basalt", OreDictUnifier.get(stone, Basalt, 1), "PP", "PP", 'P',
+                new ItemStack(ExNihiloModule.GTPebbles, 1, 0));
+        ModHandler.addShapedRecipe("black_granite", OreDictUnifier.get(stone, GraniteBlack, 1), "PP", "PP", 'P',
+                new ItemStack(ExNihiloModule.GTPebbles, 1, 1));
+        ModHandler.addShapedRecipe("marble", OreDictUnifier.get(stone, Marble, 1), "PP", "PP", 'P',
+                new ItemStack(ExNihiloModule.GTPebbles, 1, 2));
+        ModHandler.addShapedRecipe("red_granite", OreDictUnifier.get(stone, GraniteRed, 1), "PP", "PP", 'P',
+                new ItemStack(ExNihiloModule.GTPebbles, 1, 3));
+
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ModItems.pebbles, 4, 0))
+                .output(Blocks.COBBLESTONE)
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ModItems.pebbles, 4, 1))
+                .outputs(new ItemStack(Blocks.STONE, 1, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ModItems.pebbles, 4, 2))
+                .outputs(new ItemStack(Blocks.STONE, 1, 3))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ModItems.pebbles, 4, 3))
+                .outputs(new ItemStack(Blocks.STONE, 1, 5))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 0))
+                .outputs(OreDictUnifier.get(stone, Basalt, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 1))
+                .outputs(OreDictUnifier.get(stone, GraniteBlack, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 2))
+                .outputs(OreDictUnifier.get(stone, Marble, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 3))
+                .outputs(OreDictUnifier.get(stone, GraniteRed, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+
+        // Crushed stones
```

Comment:
Do you want to add a recipe here for crushed skystone as well, behind a check for if AE2 is loaded?